blueprints/showbie-tooling/files/.editorconfig

```diff
@@ -1,23 +1,25 @@
-# EditorConfig helps developers define and maintain consistent
-# coding styles between different editors and IDEs
 # editorconfig.org
-
 root = true
 
 [*]
-end_of_line = lf
 charset = utf-8
-trim_trailing_whitespace = true
-insert_final_newline = true
-indent_style = space
+end_of_line = lf
 indent_size = 2
-max_line_length = 100
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.{js,css,scss}]
+max_line_length = 80
 
 [*.hbs]
 insert_final_newline = false
 
 [*.{diff,md}]
 trim_trailing_whitespace = false
 
+[*.{yml,yaml}]
```

Comment:
Personally I think we can just remove this. Firstly, it really seems harder to read YML with 4 spaces instead of 2 and secondly, this is just extra config that the `[*]` selector above can cover.

Reply:
Mm, agree to disagree on that first point 😉 I can never remember how YAML translates anyway so I'm fine to revert this and reformat the .yml files in the blueprint. The `[*]` selector would have set all file formats to 4-space indents, instead of just YAML files.

Reply:
Should I remove the `[Makefile]` selector as well?

Reply:
Regarding indentation, I just meant that we have all files (`[*]`) set to 2 spaces via `indent_size`. Removing the yml specific rule here would enforce 2 spaces in `yml` files.

Reply:
Don't remove the `Makefile` rule because they explode if tabs are not used. Granted, we no longer have a Makefile 🤷‍♂️